src/test/java/org/jitsi/meet/test/StartMutedTest.java

```diff
@@ -57,7 +57,16 @@ public void checkboxesTest()
         driver1.findElement(By.id("startAudioMuted")).click();
         driver1.findElement(By.id("startVideoMuted")).click();
 
-        ensureTwoParticipants();
+        WebParticipant participant = joinSecondParticipant();
+
+        // if the participant is audio only, if audio is muted we will not
+        // receive any data, so skip download check
+        // if the participant is audio only and audio muted there will be no
+        // data sent
+        participant.waitForSendReceiveData(
+            !participant.isAudioOnlyParticipant(),
+            !participant.isAudioOnlyParticipant());
```

Comment:
shouldn't it be:
`participant2.waitForSendReceiveData(`
`    !participant2.isAudioOnlyParticipant(),`
`    !participant1.isAudioOnlyParticipant());`
?

Reply:
I think I had fixed that and force pushed the second commit of the PR.